### ISSUE TITLE
256 pixels resolution not working, blank images generated

### ISSUE THREAD

**Issue by nikjetchev** [AaronLeong/BigGAN-pytorch#13]
I am training on my custom dataset, which works really good with the default 128 pixels resolution.

I tried setting imsize=256, and using hinge loss.
The generated images are always blank white.
When looking at the program output, the values of the discriminator for both true and generated images are always 0.0 , so obviously something does not work right. 

My hypothesis: the fixed count of layers in the generator and discriminator are set for 128 pixels resolution, will adding more conv blocks at some place fix the behaviour for 256 pixels resolution?

thanks a lot for the help

**Comment by darknoon** [comment]
Yes, I have encountered the same issue. In my case, moving to 512x512, I needed to make the following changes:

- add additional residual blocks to get to 7 total (follow description on p18 of the paper)
- split Z into 16-entry chunks instead of 20 (so we have 8 available)
- change instances of 148 (128 embedding size + 20 chunk size) to 144 (128 + 16 chunk size)

**Comment by blackcat84** [comment]
You can have a look at my fork, where I've introduced support for 512x512 following the paper and @darknoon suggestions (disclaimer: I'm not very experienced with GANs/images so there's a non-negligible chance of bugs/something conceptually wrong).

**Comment by jakubLangr** [comment]
Oddly I have been getting a massive explosion in the losses even at 128x128 (custom dataset). So I reckon something is wrong.
![Screen Shot 2019-10-04 at 12 03 59 PM](https://user-images.githubusercontent.com/1884004/66203004-164f6000-e69f-11e9-9404-4b20b5994386.png)

**Comment by Lihengda-follower** [comment]
> 我正在对自定义数据集进行训练，该数据集在默认的128像素分辨率下效果很好。
> 
> 我尝试设置imsize = 256，并使用铰链损耗。
> 生成的图像始终为空白白色。
> 当查看程序输出时，真实图像和生成图像的鉴别器的值始终为0.0，因此显然某些操作不正确。
> 
> 我的假设：将生成器和鉴别器中固定的层数设置为128像素分辨率，是否会在某个位置添加更多转换块来解决256像素分辨率的问题？
> 
> 非常感谢您的帮助

您好，我想咨询您是用的什么配置的gpu，1050完全不能够运行。
